README.md

```diff
@@ -26,9 +26,9 @@ More information about these can be found [here](https://github.com/Azure/autore
 3. Hook up plugins into the AutoRest pipeline DAG, e.g.
 
 ```yaml
-version: 3.0.6267
+version: 3.0.6320

```

Comment:
For my own edification, should I bump this number every time I add a new feature?

Reply:
We have been updating this on a need-to basis, so usually when there are new features or fixes in Autores.Core or ModelerFour or new tests when bumping test-server. In this particular case I needed this bump for a fix in Autorest.Core for supporting `x-ms-text` since TestServer started using that extension generating swaggers with it was failing.

I also try to get modeler four to the latest version once a month to be sure we keep up. In the near future we'd probably point to the latest builds within the same major build.

Reply:
I see, thanks!